### DISCUSSION TITLE
Tour + photos

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1132] [General]
HI
After a tour is selected in 2D Map, it disappears when picking one or several photos shot during the tour!
What did I do wrong?

**Comment by wolfgang-ch** [comment]
> it disappears 

What disappears?

**Comment by gringojack** [comment]
The GPS trace!

**Comment by wolfgang-ch** [comment]
When I try to do what you described, then the tour track do not disappear

https://github.com/mytourbook/mytourbook/assets/1283445/99c8e7c7-738d-436d-aae4-05cd67d636b0

**Reply by gringojack** [reply]
If I click on this icon, track will disappear. If I click back on it, it wont reappear. Is that normal?

![myexample](https://github.com/mytourbook/mytourbook/assets/4082252/4f0bbdbe-4abe-4b21-b502-5da26085068b)

**Comment by wolfgang-ch** [comment]
I do not understand where you click and when the track disappears. 

Can you create a video so I can see the steps you are doing.
